resources/js/common.js

```diff
@@ -283,3 +282,40 @@ function postwith (to,p) {
   document.body.removeChild(myForm) ;
 }
 
+
+function isValidDate(dateString)

```

Comment:
Instead of writing our own date validation code, we should consider using jQuery. Note: CORAL currently uses jQuery 1.4.4.  Here is an example:
http://stackoverflow.com/questions/6598386/jquery-validation-date